socket.c

```diff
@@ -27,13 +27,14 @@
   #endif
 #endif
 
-cSatipSocket::cSatipSocket()
+cSatipSocket::cSatipSocket(size_t rcvBufSizeP)
 : socketPortM(0),
   socketDescM(-1),
   isMulticastM(false),
   useSsmM(false),
   streamAddrM(htonl(INADDR_ANY)),
-  sourceAddrM(htonl(INADDR_ANY))
+  sourceAddrM(htonl(INADDR_ANY)),
+  rcvBufSizeM(rcvBufSizeP)
```

Comment:
Let's get rid of constructor parameter and simply use: ```rcvBufSizeM(SatipConfig.GetRtpRcvBufSize())```

Reply:
That would imply that e.g. the RTCP socket uses the same buffer size even if there are no overruns on this socket. Might not be much, but still that would be a waste of resources, don` t you think?

Reply:
Fair enough, so let's leave it as is.

Reply:
IMO, it would be a bit more clearer, if we had a two constructors: one without parameters and another with buffer size.

Reply:
I have added the second constructor. I was not sure though, whether we could use C++11 constructor delegation, so I just copied the initialization of the other members to keep the code compatible.